specification/monitor/resource-manager/Microsoft.Insights/preview/2021-05-01-preview/diagnosticsSettingsCategories_API.json

```diff
@@ -94,7 +94,7 @@
           "default": {
             "description": "Error response describing why the operation failed.",
             "schema": {
-              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+              "$ref": "#/definitions/ErrorResponse"
```

Comment:
suggest not revert this to fix linter,  2 reasons
1. if consider breaking change. actually you have also other breaking change.
2. And also to keep consistent , as I saw they use  common definition in other swagger files